src/unstract/sdk/adapters/x2text/llm_whisperer/src/static/json_schema.json

```diff
@@ -83,6 +83,12 @@
       "default": "",
       "pattern": "^(\\s*\\d+-\\d+|\\s*\\d+-|\\s*\\d+|^$)(,\\d+-\\d+|,\\d+-|,\\d+)*$",
       "description": "Specify the range of pages to extract (e.g., 1-5, 7, 10-12, 50-). Leave it empty to extract all pages."
+    },
+    "page_seperator": {
```

Comment:
@kirtimanmishrazipstack Are we not adding the regex pattern?

Reply:
@Deepak-Kesavan had a discussion with @arun-venkataswamy , we won't be going with regex for `page_separator`.  In code we will just find and replace `{{page_no}}`